.gitignore

```diff
@@ -44,6 +44,4 @@ website/node_modules
 website/i18n/*
 !website/i18n/en.json
 
-project/metals.sbt
-.bloop/*
-.metals/*
\ No newline at end of file
+project/metals.sbt
```

Comment:
Should this change be included?

Reply:
That's fine, we want to have it gitignored